### PR TITLE
Add array-type linter configuration

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

File: packages/faucet/src/faucet.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export class Faucet {
     apiUrl: string,
     addressPrefix: string,
     config: TokenConfiguration,
-    pens: readonly [string, Pen][],
+    pens: ReadonlyArray<[string, Pen]>,
```
Comment (willclarktech) [inline packages/faucet/src/faucet.ts:41]: Now can we have `ReadonlyArray<readonly [string, Pen]>`?

File: .eslintrc.js [hunk 2]
```diff
@@ -27,10 +27,11 @@ module.exports = {
     "no-shadow": "warn",
     "no-unused-vars": "off", // disabled in favour of @typescript-eslint/no-unused-vars, see https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md
     "prefer-const": "warn",
-    "radix": ["warn", "always"],
+    radix: ["warn", "always"],
     "spaced-comment": ["warn", "always", { line: { markers: ["/ <reference"] } }],
     "import/no-cycle": "warn",
     "simple-import-sort/sort": "warn",
+    "@typescript-eslint/array-type": ["warn", { default: "array-simple" }],
```
Comment (willclarktech) [inline .eslintrc.js:34]: This seems like the most confusing configuration, but I'm OK with it.

File: packages/faucet/src/faucet.ts (hunk 1 above)
Comment (webmaster128) [reply packages/faucet/src/faucet.ts:41]: Indeed, we can

File: .eslintrc.js (hunk 2 above)
Comment (webmaster128) [reply .eslintrc.js:34]: I don't have a strong opinion here. This was used in the React projects if I recall correctly and worked well.

**Pull request merged**